packages/react-core/src/components/SearchInput/SearchInput.tsx

```diff
@@ -277,11 +285,9 @@ const SearchInputBase: React.FunctionComponent<SearchInputProps> = ({
 
   if (!!onSearch || attributes.length > 0 || !!onToggleAdvancedSearch) {
     if (attributes.length > 0) {
-      return (
-        <div className={css(className, styles.searchInput)} ref={searchInputRef} {...props}>
-          {buildSearchTextInputGroupWithExtraButtons()}
+      const AdvancedSearch = (
+        <span>
           <AdvancedSearchMenu
-            className={styles.searchInputMenu}
```

Comment:
Should this class be getting removed? Based on the screenshots provided in the issue I believe that this class was present before the bug was introduced.

Reply:
yes,
we changed the implementation using the popper and we don't need to these class to set the right style/position

Reply:
Got it, thanks!